README.md

```diff
@@ -168,26 +168,45 @@ cp solr_configs/conf/schema.xml solr_scripts/schema.xml
 
 The Dockerized versions of Solr on the server need your attention, too. Follow these steps.
 
-1. Deploy the schema changes on the staging server. 
-2. Shell into the server, go to the `lametro-staging` repo, and remove and rebuild the Solr container.
+1. Deploy the schema changes on the staging server.
+2. Shell into the server as the ubuntu user, go to the `lametro-staging` repo, and remove and rebuild the Solr container.
```

Comment:
I think it would be more instructive to break this into two steps, i.e.:

1. Shell in the Metro server, and go to `lametro-staging`.

```
ssh ubuntu@boardagendas.metro.net

# Do not switch to the DataMade user!
# You need sudo privileges to run docker commands
cd /home/datamade/lametro-staging
```

2. Remove and rebuild the Solr container.

```
...
```